graphics/dmtx-utils/Portfile

```
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        dmtx dmtx-utils 0.7.6 v
+categories          graphics
+platforms           darwin
+maintainers         nomaintainer
+license             LGPL-2.1
+
+description         Data Matrix command line utilities
+
+long_description    libdmtx is a software library that enables programs to \
+                    read and write Data Matrix barcodes of the modern ECC200 \
+                    variety. This package, dmtx-utils, provides command line \
+                    utilities that allow scripts and command line users to use \
+                    libdmtx functionality
+
+use_autoreconf      yes
+
+checksums           rmd160  25a1827964757d38fce8022a196718bd4ce248bc \
+                    sha256  14e22455473d4da1d131307015464a82a03b88e25c5285024b2a2840f595873d \
+                    size    54898
+
+depends_build       port:pkgconfig port:autoconf
```

Just nitpicking (newline and alphabetical order):
```suggestion
depends_build       port:autoconf \
                    port:pkgconfig
```

---

That's really useful to know, thank you for the clarification.